portfolio/src/main/webapp/index.html

```diff
@@ -2,20 +2,23 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Steph Walsh</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <!-- TODO: sticky header --> 
-    <h1 class="header">Steph Walsh</h1> 
 
+    <!-- TODO: sticky header --> 
+    <header class="header">
```

Comment:
Nit: You could refer to this element with `header` instead of `.header` and remove the class attribute entirely.